godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -49,6 +50,16 @@ kotlin {
     }
 }
 
+val generateAPI by tasks.creating(GenerateApiTask::class) {
+    source.set(project.file("$rootDir/godot-kotlin/godot-headers/api.json"))
+    outputDirectory.set(project.file("$rootDir/godot-kotlin/godot-library/src/nativeGen/kotlin/"))
+}
+
+tasks {
```

Comment:
A more type-safe approach is to use `tasks.withType<KotlinCompile> { ... }`

Reply:
Done ! :)